script.js

```diff
@@ -1,6 +1,243 @@
+        console.log("Time started")
+        timer = Date.now();
+    }
+
+    if(typing.textContent === firstQuote.q){
+        // END A TIMER:
+        const elapsedTime = Date.now() - timer;
+        console.log('Elapsed Time:', elapsedTime + 'ms');
+        wordsPerMinuteCalc(elapsedTime);
+    }
+}
+
+function wordsPerMinuteCalc(elapsedTime){
+    let charecters = firstQuote.c;
+    let timeTaken = (elapsedTime / 1000) / 60
+    console.log("charecters: ", charecters, "misstypes: ", missTypes, "time Taken:", timeTaken);
+    let WPM = ((charecters / 5) - missTypes) / timeTaken;
+    WPM = Math.round(WPM);
+    if (WPM <= 0){
+        WPM = 0;
+    }
+    console.log(`Words per minute: ` + WPM);
+
+
+    WPMText = document.querySelector('#WPM');
+    WPMText.textContent = `Words Per Minute: ` + WPM;
+}
+
+function switchTextColor(){
+    // let switchColorBtn = document.querySelector('#switchTextColor');
+    console.log('switchTextColor is being run!')
+
+    const typing = document.querySelector('#typing'); 
+    const p1 = document.querySelector('#quote');
+    const p2 = document.querySelector('#author');
+    const p3 = document.querySelector('#charecters');
+    const p4 = document.querySelector('#WPM');
+    const p5 = document.querySelector('#userName');
+    const p6 = document.querySelector('#title');
+    const p7 = document.querySelector('#clear-local-storage');
+    const p8 = document.querySelector('#switchTextColor');
+    const p9 = document.querySelector('#reload');
+
+    localStorage.getItem('typing');
+    localStorage.getItem('p1');
+    localStorage.getItem('p2');
+    localStorage.getItem('p3');
+    localStorage.getItem('p4');
+    localStorage.getItem('p5');
+    localStorage.getItem('p6');
+    localStorage.getItem('p7');
+    localStorage.getItem('p8');
+    localStorage.getItem('p9');
+
+    if(typing.style.color === 'white'){
+        typing.style.color = 'black';
+        typing.style.borderColor = 'black';
+        p1.style.color = 'black';
+        p2.style.color = 'black';
+        p3.style.color = 'black';
+        p4.style.color = 'black';
+        p5.style.color = 'black';
+        p6.style.color = 'black';
+        p7.style.color = 'black';
+        p7.style.borderColor = 'black';
+        p8.style.color = 'black';
+        p8.style.borderColor = 'black';
+        p9.style.color = 'black';
+        p9.style.borderColor = 'black';
+    }
+    else{
+        typing.style.color = 'white';
+        typing.style.borderColor = 'white';
+        p1.style.color = 'white';
+        p2.style.color = 'white';
+        p3.style.color = 'white';
+        p4.style.color = 'white';
+        p5.style.color = 'white';
+        p6.style.color = 'white';
+        p7.style.color = 'white';
+        p7.style.borderColor = 'white';
+        p8.style.color = 'white';
+        p8.style.borderColor = 'white';
+        p9.style.color = 'white';
+        p9.style.borderColor = 'white';
+    }
+}
```

Comment:
This was a cool bit of exploration that you did with colors and local storage too!